CONTRIBUTING.md

```diff
@@ -1,11 +1,78 @@
-# Contributing
+# Overview
 
-**Technical Steering Committee (TSC)**
+This project aims to be governed in a transparent, accessible way for the benefit of the community. All participation in this project is open and not bound to corporate affilation. Participants all our bound to the ASWF Code of Conduct.
+
+# Project roles
+
+## Contributor
+
+The contributor role is the starting role for anyone participating in the project and wishing to contribute code.
+
+# Process for becoming a contributor
+
+* Review the [coding standards](http://www.openvdb.org/documentation/doxygen/codingStyle.html) to ensure your contribution is in line with the project's coding and styling guidelines.
+* Sign the Individual CLA and have your organization sign the Corporate CLA.
+* Submit your code as a PR with the appropriate DCO signoff.
+* Have your submission approved by the maintainer(s) and merged into the codebase.
+
+## Committer
+
+The committer role enables the participant to commit code directly to the repository, but also comes with the obligation to be a responsible leader in the community.
+
+### Process for becoming a committer
+
+* Show your experience with the codebase through contributions and engagement on the community channels.
+* Request to become a committer.
+* Have the majority of committers approve you becoming a committer.
+* Your name and email is added to the MAINTAINERS.md file for the project.
```

Comment:
Minor quibble - perhaps "committers" should be listed instead in COMMITTERS.md or otherwise all references to "committers" changed to "maintainers" for improved consistency?

Reply:
Good catch - this can easily be changed to COMMITTERS.md, or added to the end of this file if that's easier.

Reply:
Sounds good.